src/main/java/tric/tricproject/security/jwt/JwtUtils.java

```diff
@@ -0,0 +1,56 @@
+package tric.tricproject.security.jwt;
+
+import io.jsonwebtoken.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Component;
+import tric.tricproject.security.services.UserDetailsImpl;
+
+import java.util.Date;
+
+@Component
+public class JwtUtils {
+	private static final Logger logger = LoggerFactory.getLogger(JwtUtils.class);
+
+	@Value("${bezkoder.app.jwtSecret}")
+	private String jwtSecret;
+
+	@Value("${bezkoder.app.jwtExpirationMs}")
```

Comment:
Can you please change the names of these values?(maybe tric.app.jwtSecret)? 
Don't you need to add them to 'application.properties' file?